### ISSUE TITLE
PTN002476009  

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#1938]
"regulation of microtubule based process" from a single human annotation seems very weak.
I've been following this protein for a long time, it is one of the few unconserved things conserved from bacteria to human, and so it must have a fundamental role (which is likely conserved).

I thought PAINT annotation transfer required some sort of triangulation from 2 nodes?

**Comment by pgaudet** [comment]
> I thought PAINT annotation transfer required some sort of triangulation from 2 nodes?
No. Ideally there is 'enough' support but a single annotation is sufficient to allow transfer.

**Comment by ValWood** [comment]
"regulation of microtubule process" seems odd here.
I think MEMO1  is likely "signalling" or "transcription".

We know from a number of sources that it interacts with
http://www.uniprot.org/uniprot/Q9Y4X0
causative gene for Alport syndrome, but we don't know what this does either...
There is some evidence that they interact with Mediator complex....
but both proteins are also conserved in bacteria which is strange for signaling....
